R/dfm.R

```diff
@@ -354,17 +365,17 @@ dfm.dfm <- function(x,
     language <- "english"
     if (stem) {
         if (verbose) catm("   ... stemming features (", stri_trans_totitle(language), ")", sep="")
-        oldNfeature <- nfeature(x)
+        n_old <- nfeature(x)
         x <- dfm_wordstem(x, language)
         if (verbose) 
-            if (oldNfeature - nfeature(x) > 0) 
-                catm(", trimmed ", oldNfeature - nfeature(x), " feature variant",
-                     ifelse(oldNfeature - nfeature(x) != 1, "s", ""), "\n", sep = "")
+            if (n_old - nfeature(x) > 0) 
+                catm(", trimmed ", n_old - nfeature(x), " feature variant",
+                     ifelse(n_old - nfeature(x) != 1, "s", ""), "\n", sep = "")
     }
     
     # remove any NA named columns
-    if (any(naFeatures <- is.na(featnames(x))))
-        x <- x[, -which(naFeatures), drop = FALSE]
+    if (any(is.na(featnames(x))))
```

Comment:
Just made code simpler, because I did not think this is performance critical (there should be no NA in columns). You can keep current code.